.golangci.yml

```diff
@@ -31,6 +30,51 @@ linters-settings:
       - standard
       - default
       - prefix(github.com/testcontainers)
+  nakedret:
+    max-func-lines: 0
+  revive:
```

Comment:
suggestion: This looks like the defaults, it's cleaner to override any settings we want to so we should be able to remove this.